src/tests/outline.test.ts

```diff
@@ -0,0 +1,214 @@
+import { describe, test, expect } from '@jest/globals';
+
+import { SVGPathData } from '../SVGPathData.js';
+
+import { OUTLINE } from '../transformers/outline/outline.js';
+import type { OffsetOptions } from '../transformers/outline/options.js';
+
+// Group options by type
+const TEST_CONFIGS = {
+  // Basic width tests for all shapes
+  widthTests: [{ width: 1 }, { width: 2 }, { width: 5 }, { width: 10 }],
+
+  // Line cap variations for open paths
+  lineCapTests: [
+    { width: 3, linecap: 'butt' },
+    { width: 3, linecap: 'square' },
+    { width: 3, linecap: 'round' },
+  ],
+
+  // Line join variations for closed paths
+  lineJoinTests: [
+    { width: 3, linejoin: 'miter' },
+    { width: 3, linejoin: 'bevel' },
+    // Add new tests for round joins
+    { width: 3, linejoin: 'round' },
+    { width: 3, linejoin: 'miter-clip' },
+  ],
+
+  // Miter limit tests for shapes with corners
+  miterLimitTests: [
+    { width: 3, linejoin: 'miter', miterLimit: 2 },
+    { width: 3, linejoin: 'miter', miterLimit: 8 },
+    { width: 3, linejoin: 'miter-clip', miterLimit: 2 },
+    { width: 3, linejoin: 'miter-clip', miterLimit: 8 },
+    { width: 3, linejoin: 'round', miterLimit: 2 },
+    { width: 3, linejoin: 'round', miterLimit: 8 },
+  ],
+
+  // Combined option tests (more complex combinations)
+  combinedTests: [
+    { width: 4, linecap: 'butt', linejoin: 'miter', miterLimit: 4 },
+    { width: 4, linecap: 'square', linejoin: 'bevel' },
+    { width: 4, linecap: 'square', linejoin: 'round' },
+    { width: 4, linecap: 'round', linejoin: 'miter-clip' },
+  ],
+} satisfies Record<string, Partial<OffsetOptions>[]>;
+
+const TEST_SHAPES = {
```

Comment:
No arcs?

Reply:
I see generateOutlinePath handles that by doing .aToC, but it's not being validated.

Reply:
yes current implementation supports only lines and beziers, this def should/could be extended but as POC, i didn't bother